insights/parsers/tests/test_teamdctl_state_dump.py

```diff
@@ -0,0 +1,59 @@
+from insights.parsers.teamdctl_state_dump import TeamdctlStateDump
+from insights.tests import context_wrap
+
+TEAMDCTL_STATE_DUMP_INFO = """
+{
+    "runner": {
+        "active_port": "eno1"
+    },
+    "setup": {
+        "daemonized": false,
+        "dbus_enabled": true,
+        "debug_level": 0,
+        "kernel_team_mode_name": "activebackup",
+        "pid": 4464,
+        "pid_file": "/var/run/teamd/team0.pid",
+        "runner_name": "activebackup",
+        "zmq_enabled": false
+    },
+    "team_device": {
+        "ifinfo": {
+            "dev_addr": "2c:59:e5:47:a9:04",
+            "dev_addr_len": 6,
+            "ifindex": 5,
+            "ifname": "team0"
+        }
+    }
+}
+""".strip()
+
+
+class TestTeamdctlStateDump():
+    def test_teamdctl_state_dump(self):
+        result = TeamdctlStateDump(context_wrap(TEAMDCTL_STATE_DUMP_INFO))
```

Comment:
You could implement this as test functions instead of adding the complexity of a class, unless you need the class for some other reason.